week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,17 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+function tidyUpString(strArr) {
+  let tidied = [];
+    function tidy(name) {
```

Comment:
Very nice idea here to have a separate function called `tidy` - you just need to make sure you are indenting your code so it is a bit easier to read. Whenever you start a new function you should move 2 spaces into the right so others will see that this code is part of the function